### PR TITLE
Adds an admin verb for turning all lights on or off

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Adds an admin verb for turning all lights on or off

## Why It's Good For The Game
Allows for more adminbus + for easier local testing when messing around with lighting.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/110184118/d0802986-77ff-4aff-a06e-dd2162d92a65

</details>

## Changelog
:cl:
admin: Added a new verb for toggling all lights on or off, found in the Secrets panel
/:cl:

### CODE REVIEW

File: code/modules/admin/secrets.dm [hunk 1]
```diff
@@ -193,7 +194,23 @@ GLOBAL_DATUM_INIT(admin_secrets, /datum/admin_secrets, new)
 				if("Off")
 					SSnightshift.can_fire = FALSE
 					SSnightshift.update_nightshift(FALSE, TRUE)
-
+		if("all_light_toggle")
+			if(!check_rights(R_ADMIN))
+				return
+			var/val = alert(usr, "Do you want to turn all lights on or off?", "Light Manipulation", "On", "Off")
```
Comment (EvilDragonfiend) [inline code/modules/admin/secrets.dm]: the selection option should be 3: on, off, cancel.

File: code/modules/admin/secrets.dm [hunk 2]
```diff
@@ -193,7 +194,23 @@ GLOBAL_DATUM_INIT(admin_secrets, /datum/admin_secrets, new)
...
+			var/set_to = null
+			switch(val)
+				if("On")
+					set_to = TRUE
+				if("Off")
+					set_to = FALSE
```
Comment (EvilDragonfiend) [inline code/modules/admin/secrets.dm:208]: put an early return for cancel option.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Review by EvilDragonfiend — APPROVED** [review]

File: code/modules/admin/secrets.dm [hunk 3]
```diff
@@ -193,7 +194,25 @@ GLOBAL_DATUM_INIT(admin_secrets, /datum/admin_secrets, new)
 				if("Off")
 					SSnightshift.can_fire = FALSE
 					SSnightshift.update_nightshift(FALSE, TRUE)
-
+		if("all_light_toggle")
+			if(!check_rights(R_ADMIN))
+				return
+			var/val = alert(usr, "Do you want to turn all lights on or off?", "Light Manipulation", "On", "Off", "Cancel")
+			var/set_to = null
+			switch(val)
```
Comment (itsmeow) [inline code/modules/admin/secrets.dm:202]: The default case is not handled here

**Comment by Dejaku51** [comment]
just learn sdql smh

**Review by itsmeow — APPROVED** [review]

**Pull request merged**